DataFormats/EgammaCandidates/src/classes_def.xml

```diff
@@ -176,7 +176,21 @@
    <version ClassVersion="11" checksum="1799511018"/>
    <version ClassVersion="10" checksum="4260674990"/>
   </class>
-  <class name="reco::GsfElectron::ConversionRejection" ClassVersion="11">
+  <class name="reco::GsfElectron::ConversionRejection" ClassVersion="12">
+   <version ClassVersion="12" checksum="2742251964"/>
+  <ioread sourceClass="reco::GsfElectron::ConversionRejection" version="[1-11]" source="reco::TrackBaseRef partner" targetClass="reco::GsfElectron::ConversionRejection" target="partner">
+    <![CDATA[
+    auto& ref = onfile.partner;
+    edm::EDProductGetter const* getter = ref.productGetter();
+    if (getter) {
+      std::cout<<"yes getter\n";
+      partner = reco::TrackPtr(ref.id(), ref.key(), getter);
+    } else {
+      std::cout<<"no getter\n";
+      partner = reco::TrackPtr(ref.id(), ref.get(), ref.key());
```

Comment:
This path is taken, and in the parent function a `free(): invalid pointer` is raised

Reply:
best not to put debugging output into a PR :). Can you get a stack trace for the failure?